site/realm/functions/config.json

```diff
@@ -43,6 +43,22 @@
         "name": "logReportHistory",
         "private": false
     },
+    {
+        "name": "onNewEntryFunction_aiidprod",
+        "private": true
+    },
+    {
+        "name": "onNewEntryFunction_customData",
+        "private": true
+    },
+    {
+        "name": "onNewEntryFunction_history",
+        "private": true
+    },
+    {
+        "name": "onNewEntryFunction_translations",
+        "private": true
+    },
```

Comment:
Add the `setCreatedAtField` function definition in this config file as well. To avoid this missing part in the future make sure you pull the code from Atlas using `realm-cli pull` command and check the code differences